pkg/v1/providers/infrastructure-byoh/v0.1.0/ytt/base-template.yaml

```diff
@@ -0,0 +1,197 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: '${ CLUSTER_NAME }'
+  namespace: '${ NAMESPACE }'
+  labels:
+    tkg.tanzu.vmware.com/cluster-name: '${ CLUSTER_NAME }'
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+      - '${ CLUSTER_CIDR}'
+    services:
+      cidrBlocks:
+      - '${ SERVICE_CIDR }'
+  controlPlaneRef:
+    apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: KubeadmControlPlane
+    name: '${ CLUSTER_NAME }-control-plane'
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+    kind: VSphereCluster
+    name: '${ CLUSTER_NAME }'
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: ByoCluster
+metadata:
+  name: '${ CLUSTER_NAME }'
+  namespace: '${ NAMESPACE }'
+spec:
+  controlPlaneEndpoint:
+    host: '${ VSPHERE_CONTROL_PLANE_ENDPOINT }'
+    port: 6443
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: ByoMachineTemplate
+metadata:
+  name: '${ CLUSTER_NAME }-control-plane'
+  namespace: '${ NAMESPACE }'
+spec:
+  template:
+    spec: {}
+---
+apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+kind: KubeadmControlPlane
+metadata:
+  name: '${ CLUSTER_NAME }-control-plane'
+  namespace: '${ NAMESPACE }'
+spec:
+  kubeadmConfigSpec:
+    useExperimentalRetryJoin: true
+    clusterConfiguration:
+      apiServer:
+        timeoutForControlPlane: "8m0s"
```

Comment:
Does 8m is enough for most of the cases?  Where does this number come from?

Reply:
For "infrastructure-aws" and "infrastructure-vsphere", they both use this value as the timeout value. For BYOH, either remove this configuration or consider a new value for this should be OK.